### PR TITLE
Extract oc and openshift-install binaries from installer image

### LINKED ISSUE #23: Extract the `openshift-install` and `oc` binaries from the installer image
Get the the openshift-install and oc binaries from the OCP4 installer image itself.

### PR DESCRIPTION
**Pull request by Prajyot-Parab**
Fixes #23

### CODE REVIEW

**Comment by Prajyot-Parab** [comment]
@yussufsh Please review this.

**Comment by bpradipt** [comment]
@Prajyot-Parab can you create an issue https://github.com/ppc64le/ocp4_upi_powervm/issues and mention "Fixes #<issue-number>" in the commit message

File: ansible/playbooks/roles/ocp-config/tasks/main.yaml [hunk 1]
```diff
@@ -1,5 +1,54 @@
...
+        chdir: "{{ tools_dir }}"
+
+    - name: Remove exsisting binaries
+      file:
+        path: "/usr/local/bin/{{ item }}"
+        state: absent
+        mode: '0755'
+      with_items:
+      - oc
+      - openshift-install
```
Comment (bpradipt) [inline ansible/playbooks/roles/ocp-config/tasks/main.yaml]: Can't we just overwrite the existing ones with new binaries and avoid creating another set of directories ?

File: ansible/playbooks/roles/ocp-config/tasks/main.yaml [hunk 2]
```diff
@@ -1,5 +1,54 @@
...
+        path: "{{ tools_dir }}"
+        state: "{{ item }}"
+        mode: '0755'
+      with_items:
+      - absent
+      - directory
+
+    - name: Extract OCP4 tools from release image
+      shell: |
+        echo '{{ install_config.pull_secret }}' > pull_secret
```
Comment (bpradipt) [inline ansible/playbooks/roles/ocp-config/tasks/main.yaml:20]: Is this step really needed? Can the pull_secret input be directly used without copying it to another file ?

Comment (Prajyot-Parab) [reply ansible/playbooks/roles/ocp-config/tasks/main.yaml:20]: The `oc adm` command parameter expects a path to pull secret file `--registry-config='': Path to your registry credentials` hence above approach

**Review by bpradipt — APPROVED** [review]
lgtm

**Pull request merged**